### PR TITLE
Navigation bar with 5 fragments

### PR DESCRIPTION
**Pull request by jenyangk**
Bunch of stuff related to nav bar and fragments

### CODE REVIEW

**Review by HonestPretzels — CHANGES_REQUESTED** [review]
Build Failed

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MainActivity.java [hunk 1]
```diff
@@ -1,13 +1,31 @@
 package io.github.cmput301f19t19.legendary_fiesta;
 
-import androidx.appcompat.app.AppCompatActivity;
 import android.os.Bundle;
 
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+
```
Comment (HonestPretzels) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/MainActivity.java]: Why is there whitespace here?

File: app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/AddPostFragment.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import androidx.fragment.app.Fragment;
+
+import io.github.cmput301f19t19.legendary_fiesta.R;
+
+public class AddPostFragment extends Fragment {
+
+
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             ViewGroup container, Bundle savedInstanceState) {
+
```
Comment (HonestPretzels) [inline app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/AddPostFragment.java:18]: Whitespace

**Review by HonestPretzels — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_add_post.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <TextView
+        android:id="@+id/text_notifications"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:textAlignment="center"
+        android:textSize="20sp"
+        android:text="Add a Mood Event"
```
Comment (liushuyu) [inline app/src/main/res/layout/fragment_add_post.xml]: Use translatable strings

File: app/src/main/res/layout/fragment_friends_moods.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    <TextView
+        android:id="@+id/text_dashboard"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:textAlignment="center"
+        android:textSize="20sp"
+        android:text="Friends' Mood Events"
```
Comment (liushuyu) [inline app/src/main/res/layout/fragment_friends_moods.xml]: ditto as above

File: app/src/main/res/layout/fragment_own_moods.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    <TextView
+        android:id="@+id/text_home"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:textAlignment="center"
+        android:textSize="20sp"
+        android:text="My Mood Events"
```
Comment (liushuyu) [inline app/src/main/res/layout/fragment_own_moods.xml]: ditto

Comment (jenyangk) [reply app/src/main/res/layout/fragment_own_moods.xml]: fixed

**Review by HonestPretzels — APPROVED** [review]

**Pull request merged**